### PR TITLE
add recommended production deploy resolution questions

### PR DESCRIPTION
**Pull request by sw-yx**
## What was changed

add recommended production deploy resolution questions

## Why?

it will take us FOREVER to figure out how to document this  - this is a cheap and easy way to raise the visibility and ensure we keep it around once we figure it out.

### CODE REVIEW

File: docs/server/production-deploy.md [hunk 1]
```diff
@@ -54,6 +54,17 @@ Use `workflow_success`, `workflow_failed`, `workflow_timeout`, `workflow_termina
 They are also include the `namespace` tag.
 Additional information is available in [this forum post](https://community.temporal.io/t/metrics-for-monitoring-server-performance/536/3).
 
+## Debugging Temporal
+
+### Debugging Temporal Server Configs
+
+Recommended techniques to be aware of for production Temporal Server setups:
+
+- Containers (tbc)
```
Comment (flossypurse) [inline docs/server/production-deploy.md:63]: What is "tbc" ?

File: docs/server/production-deploy.md [hunk 2]
```diff
@@ -54,6 +54,17 @@ Use `workflow_success`, `workflow_failed`, `workflow_timeout`, `workflow_termina
 They are also include the `namespace` tag.
 Additional information is available in [this forum post](https://community.temporal.io/t/metrics-for-monitoring-server-performance/536/3).
 
+## Debugging Temporal
+
+### Debugging Temporal Server Configs
+
+Recommended techniques to be aware of for production Temporal Server setups:
```
Comment (flossypurse) [inline docs/server/production-deploy.md:61]:
```suggestion
Recommended configuration debugging techniques for production Temporal Server setups:
```

Just a suggestion ^

**Review by flossypurse — APPROVED** [review]

File: docs/server/production-deploy.md (hunk 1 above)
Comment (sw-yx) [reply docs/server/production-deploy.md:63]: to be completed

Comment (sw-yx) [reply docs/server/production-deploy.md:63]: i'll expand a bit

File: docs/server/production-deploy.md [hunk 3]
```diff
@@ -54,6 +54,17 @@ Use `workflow_success`, `workflow_failed`, `workflow_timeout`, `workflow_termina
...
 Additional information is available in [this forum post](https://community.temporal.io/t/metrics-for-monitoring-server-performance/536/3).
 
+## Debugging Temporal
+
+### Debugging Temporal Server Configs
+
+Recommended configuration debugging techniques for production Temporal Server setups:
+
+- Containers (tbc)
+- Storage (tbc)
```
Comment (sw-yx) [inline docs/server/production-deploy.md:64]:
```suggestion
- Containers (to be completed)
- Storage (to be completed)
```

**Pull request merged**